api/client/javascript/src/client/schemas.ts

```diff
@@ -6097,8 +6125,42 @@ export interface components {
     PaymentTerms:
       | components['schemas']['PaymentTermInstant']
       | components['schemas']['PaymentTermDueDate']
-    /** @description Numeric representation of a percentage */
-    Percentage: string
+    /**
+     * Format: double
+     * @description Numeric representation of a percentage
+     *
+     *     50% is represented as 50
+     * @example 50
+     */
+    Percentage: number
+    /** @description Percentage price with spend commitments. */
+    PercentagePriceWithCommitments: {
+      /**
+       * @description The type of the price. (enum property replaced by openapi-typescript)
+       * @enum {string}
+       */
+      type: 'percentage'
+      /**
+       * Percentage
+       * @description The percentage of the markup of the price.
+       *
+       *     Examples:
+       *     - 0: the price is the same as the base price
+       *     - 100: the price is 2x the base price
+       *     - -50: the price is half of the base price
+       */
+      markupPercentage: components['schemas']['Percentage']
```

Comment:
make it optional. Also I think we should call it `markupRate` and use 1 to express 100%. Percemntage is weird without % unit.